lib/lightstep/tracer/tracer.rb

```diff
@@ -0,0 +1,374 @@
+require 'json'
```

Comment:
Yes, I think so too, but to do this you should rename the gem to follow ruby conventions.

gem 'lightstep' -> require 'lightstep' -> lightstep.rb is the root
gem 'lightstep-tracer' -> require 'lightstep-tracer' -> lightstep/tracer.rb is the root

Happy to do this if you'd like, you will just have to republish the gem.

Gem should be 'lightstep' if it will always be the one gem for all of lightstep. It should be 'lightstep-tracer' if you're going to make other gems (e.g. 'lightstep-logging', 'lightstep-profiling' ...)